app/src/main/java/com/example/backgroundapp/model/Photo.kt

```diff
@@ -0,0 +1,83 @@
+package com.example.backgroundapp.model
+
+
+import com.google.gson.annotations.SerializedName
+
+data class PhotoItem(
```

Comment:
iç içe data class tanımlamaları yapmayalım. hepsini ayıralım. ayrı classlarda olsun. gerekirse isimlendirmeleride düzenleyelim. Mesela iki farklı Links olduğu için birine başka bir isim verebiliriz.